app/facades/quotient_familial_facade.rb

```diff
@@ -2,15 +2,19 @@ class QuotientFamilialFacade
   attr_reader :quotient_familial
 
   def initialize(quotient_familial)
-    @quotient_familial = quotient_familial
+    @quotient_familial = quotient_familial.try(:with_indifferent_access)
```

Comment:
j'aime toujours pas try 👴 

mais ruby nous file une belle méthode de conversion pour Hash donc ça doit fonctionner ça:
```suggestion
    @quotient_familial = Hash(quotient_familial).with_indifferent_access
```

Reply:
J'vois pas bien ce qu'on y gagne là j'avoue

Reply:
Objectivement on gagne du code qui est plus robuste parce qu'on aura toujours un `hash` `with_indifferent_access` même s'il est potentiellement vide à moins de passer quelque chose qui soit tellement en dehors des clous que ruby ne s'en dépatouille pas (à moins que ça ne nous arrange pas mais dans ce cas y'a aussi mieux que `try`)

C'est aussi plus proche du concept de duck typing ce qui est pas mal non plus.

Et, de façon totalement subjective, c'est 'achement plus stylé 🤣

Reply:
all right